### ISSUE TITLE
Initial page | skipping it

### ISSUE THREAD

**Issue by abelardolg** [MaartenBaert/ssr#870]
Hi, 

I checked the option "Skip this page next time" but it still continues appearing.
Ubuntu 20.10 and last version of your software.

Best regards.

**Comment by abelardolg** [comment]
Hi again, 

I referred with my comment that when I press "save recording" and the last screen appears (it contains a button with "Back to the start screen"), the ssr returns to the start screen although I enabled the option "Skip this page next time". 

It should show the second screen ("input profile", "video input" and so on).

**Comment by Visne** [comment]
I agree, however this would make it impossible to ever see the very first page ever again. Maybe there should be two buttons, one for going back to the "About" page and one for going back to the second page? Or a checkbox that says "Skip about page"?

**Comment by Charlie-Ramirez-Animation-Studios-de-MX** [comment]
@abelardolg @Visne   

Hello, It's strange, could you verify and tell us what version of SSR you are using and how you installed it (Apt, PPA, Package and/or snap Manual compilation etc.)

As of the date of this message the current version (Based on the compilation via gitHub is 0.4.4)
_[Commonly the Apt versions of some distributions are not updated or some installations of Snap provide outdated versions, if it is the case try to install using the compilation method via Github, The installations of how to do it are in the project readme, They are simple only requires copy and paste in terminal]_
The problem of returning to the first page does not occur [Debian 11 - github Comp. 0.4.4 [ see Gif attached -I guess it should look animated]
![ezgif com-gif-maker](https://user-images.githubusercontent.com/64709446/160316838-ce5abdcf-1234-4c47-8d7e-74864e8554e6.gif)
]

**Comment by Visne** [comment]
@Charlie-Ramirez-Animation-Studios-de-MX Ah, seems like the Arch repo's are still on 4.4.3 because there is no tag for 4.4.4. I didn't realize. I will ask them to update.

That means this issue can be closed then, because it was fixed in 2293f06a0cfee9051b80c8f69df9fa58684408dc.

**Comment by MaartenBaert** [comment]
Apparently I never pushed the 0.4.4 tag. It's there now.
